tests/containers/docker_image.pm

```diff
@@ -28,24 +28,37 @@ use containers::urls 'get_suse_container_urls';
 use version_utils qw(get_os_release check_os_release);
 
 sub run {
-    my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
     my $runtime = "docker";
 
     install_docker_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
-    for my $iname (@{$image_names}) {
-        test_container_image(image => $iname, runtime => $runtime);
-        test_rpm_db_backend(image => $iname, runtime => $runtime);
-        build_container_image(image => $iname, runtime => $runtime);
-        if (check_os_release('suse', 'PRETTY_NAME')) {
-            test_opensuse_based_image(image => $iname, runtime => $runtime);
-            build_with_zypper_docker(image => $iname, runtime => $runtime);
-        }
-        else {
-            exec_on_container($iname, $runtime, 'cat /etc/os-release');
+    # We may test either one specific VERSION
+    # or comma-separated CONTAINER_VERSIONS
+    my $versions;
+    if (get_var('CONTAINER_VERSIONS') =~ /,/) {
```

Comment:
What if we have the variable with only 1 image defined (no comas)?

```suggestion
    $versions = get_var('CONTAINER_VERSIONS') ? get_var('CONTAINER_VERSIONS') : get_required_var('VERSION')
```

Reply:
```suggestion
     $versions = get_var('CONTAINER_VERSIONS', get_required_var('VERSION')) ;
```

Reply:
Then the `split(/,/, $versions)` will create an array containing only that one string.

Reply:
I guess https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12630#discussion_r642921333 resolved this one as well, right?

Reply:
Wow, this is actually even nicer solution!